services/scraper/user_agent.go

```diff
@@ -0,0 +1,46 @@
+package scraper
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+)
+
+var browserVersions = []string{
+	"firefox_58.0",
+	"firefox_57.0",
+	"firefox_56.0",
+	"firefox_52.0",
+	"chrome_88.0.4324.192",
+	"chrome_70.0.3538.77",
+	"chrome_65.0.3325.146",
+	"chrome_64.0.3282.0",
+}
+
+var osStrings = []string{
+	"Macintosh; Intel Mac OS X 10_10",
+	"Windows NT 10.0",
+	"Windows NT 5.1",
+	"Windows NT 6.1; WOW64",
+	"Windows NT 6.1; Win64; x64",
+	"X11; Linux x86_64",
+}
+
+const (
+	firefoxPattern = "Mozilla/5.0 (%s; rv:%s) Gecko/20100101 Firefox/%s"
+	chromePattern = "Mozilla/5.0 (%s) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/%s Safari/537.36"
+)
+
+func RandomUserAgent() string {
```

Comment:
[Hoang](https://github.com/hoangmirs/go-scraper/pull/29/files#diff-7f3a7415822031130510b41e67f66d98080de785c6c6a8a72816d160ee13f12fR57) used random agents that is from the  Colly library, it that is works we dont need to write custom agent